src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
The 'Lottery' class should not extend 'ColorSupplier'. Instead, use composition to have a 'ColorSupplier' instance within the 'Lottery' class if you need to use its functionality.